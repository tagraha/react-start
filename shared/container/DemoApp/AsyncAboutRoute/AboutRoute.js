import React from 'react';
import Helmet from 'react-helmet';

function AboutRoute() {
  return (
    <div style={{ textAlign: 'center' }}>
      <Helmet>
        <title>About</title>
      </Helmet>

      <p>
        Produced with{' '}
        <span role="img" aria-label="heart">
          ❤️
        </span>
      </p>

      <p>
        View our contributors list on our{' '}
        <a href="https://github.com/ctrlplusb/react-universally">GitHub</a>{' '}
        page.
      </p>
    </div>
  );
}

export default AboutRoute;
