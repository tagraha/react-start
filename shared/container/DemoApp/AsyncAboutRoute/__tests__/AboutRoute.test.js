/* eslint-disable import/no-extraneous-dependencies */

import React from 'react';
import { shallow } from 'enzyme';

import AboutRoute from '../AboutRoute';

describe('<AboutRoute />', () => {
  test('renders', () => {
    const wrapper = shallow(<AboutRoute />);
    expect(wrapper).toMatchSnapshot();
  });
});
