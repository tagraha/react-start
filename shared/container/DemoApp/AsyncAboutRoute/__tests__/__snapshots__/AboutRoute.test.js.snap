// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutRoute /> renders 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      About
    </title>
  </HelmetWrapper>
  <p>
    Produced with
     
    <span
      aria-label="heart"
      role="img"
    >
      ❤️
    </span>
  </p>
  <p>
    View our contributors list on our
     
    <a
      href="https://github.com/ctrlplusb/react-universally"
    >
      GitHub
    </a>
     
    page.
  </p>
</div>
`;
